features/view_specs/view_spec.feature

```diff
@@ -1,6 +1,55 @@
 Feature: view spec
 
-  View specs live in spec/views and render view templates in isolation.
+  View specs should be placed in `spec/views`,
+  and verify the content of view templates in isolation
+  (that is, without invoking a controller).
+
+  Overview
+  --------
+
+  View specs generally follow three steps:
+
+  ```ruby
+  assign(:widget, Widget.new)  # sets @widget = Widget.new in the view template
+
+  render
+
+  expect(rendered).to match(/text/)
+  ```
+
+  1. Use the `assign` method to set instance variables in the view.
+     Technically, `@widget = Widget.new` would work too,
+     but RSpec doesn't officially support this pattern.
+     (It only works as a side effect of the fact that
+     view specs include `ActionView::TestCase` behavior.
+     Be aware that it may be made unavailable in the future.)
```

Comment:
Please delete from Technically onwards, we don't want to encourage that usage. This whole step would be better thought of as "setup", e.g.
```
1. Setup up required data for the view, e.g. using the `assign` method.
```

Reply:
Done; this is [in the current version of the README](https://github.com/rspec/rspec-rails#assignkey-val), so I tried to find another place for it instead of deleting it.